lib/loader/file_loader.js

```diff
@@ -119,7 +119,7 @@ function getExports(fullpath, initializer, isCall, inject) {
 
   // process exports as you like
   if (initializer) {
-    exports = initializer(exports);
+    exports = initializer(exports, fullpath);
```

Comment:
恩，改成 opt

TZ | 天猪 <notifications@github.com>于2016年8月17日 周三上午8:25写道：

> In lib/loader/file_loader.js
> <https://github.com/eggjs/egg-core/pull/9#discussion_r75045138>:
>
> > @@ -119,7 +119,7 @@ function getExports(fullpath, initializer, isCall, inject) {
> >
> >    // process exports as you like
> >    if (initializer) {
> > -    exports = initializer(exports);
> > +    exports = initializer(exports, fullpath);
>
> 第二个参数是不是改为 info { fullpath, ... } 之类的 Object, 预留扩展位好点?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eggjs/egg-core/pull/9/files/268073f496b5cfcbf8f449b43238ba288bf7db1f#r75045138>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAWA1bt6OxIpc_5Kp9V9zsPyZkV4wHKgks5qglTwgaJpZM4JlsDo>
> .
>